### PR TITLE
Youtube element improvements

### PR DESCRIPTION
**Pull request by aschempp**

### CODE REVIEW

File: system/modules/core/elements/ContentYouTube.php [hunk 1]
```diff
@@ -78,7 +78,7 @@ protected function compile()
 
 		$objFile = new \stdClass();
 		$objFile->mime = 'video/x-youtube';
-		$objFile->path = '//www.youtube.com/watch?v=' . $this->youtube;
+		$objFile->path = 'https://www.youtube.com/watch?v=' . $this->youtube;
```
Comment (leofeyer) [inline system/modules/core/elements/ContentYouTube.php]: Why this?

Comment (aschempp) [inline system/modules/core/elements/ContentYouTube.php]: Because we should always use secure URLs if available.

Comment (Toflar) [inline system/modules/core/elements/ContentYouTube.php]: BTW: we did have this discussion already like 4 times :D

**Comment by leofeyer** [comment]
We should generally use the iframe embedding in the future (also see #8217).

**Comment by aschempp** [comment]
I still think this should make it into 3.5 …

**Comment by jscholtysik** [comment]
+1

**Comment by leofeyer** [comment]
Fixed in fd168c3a2f40df650efd41cd4e1f5009e95c90b1.

**Pull request closed without merging**